student.rb

```diff
@@ -0,0 +1,14 @@
+# student.rb
+
+require_relative 'person'
+
+class Student < Person
+  def initialize(id, age, classroom, name = 'Unknown', parent_permission: true)
+    super(id, age, name, parent_permission)
```

Comment:
- Hey, in the issue I pointed out about having the ID in the constructor above, in addition, after you've updated the code base on the suggested changes, your id will be generated automatically, bearing in my that a teacher and student is a person, the ID only needs to be retrieved from that person class, therefore, you should remove it from the super and initialize parameter.